.circleci/config.yml

```
@@ -17,6 +17,20 @@ workflows:
           matrix: &go-version-matrix
             parameters:
               go_version: ["1.16.11", "1.17.4"]
+  "nightly_build_and_test_with_algod_comparison":
```

```suggestion
  "circleci_build_and_test_nightly":
```
We'll be adding other tests too eventually, so we might want to keep the name standard.